tests/admin_views/tests.py

```diff
@@ -5801,6 +5801,34 @@ def setUp(self):
             title="A Long Title", published=True, slug="a-long-title"
         )
 
+    def test_add_group_aria_labels(self):
```

Comment:
In terms of what to test :thinking: maybe:
- the `aria-labelledby` id for the from and to box as well as the content of this (the label and help text)
- the `aria-label` for the filter boxes
- that the buttons are buttons not links